tests/Application/Actions/CreatePersonTest.php

```diff
@@ -147,8 +147,44 @@ public function testMissingData(): void
 
         $expectedJSON = json_encode([
             'error' => [
-                'description' => 'Validation error: Password is required to create an account; ' .
-                    'last_name must not be blank; email_address must not be blank',
+                'description' => 'Validation error: Password of 10 or more characters' .
+                ' is required to create an account; last_name must not be blank; email_address must not be blank',
+                'type' => 'BAD_REQUEST',
+            ],
+            'statusCode' => 400,
+        ], JSON_THROW_ON_ERROR);
+        $this->assertJsonStringEqualsJsonString($expectedJSON, $payloadJSON);
+    }
+
+    public function testTooShortPassword(): void
+    {
+        $person = $this->getTestPerson();
+
+        $app = $this->getAppInstance();
+
+        $personRepoProphecy = $this->prophesize(PersonRepository::class);
+        $personRepoProphecy->persist(Argument::type(Person::class))
+            ->shouldNotBeCalled();
+
+        $app->getContainer()->set(PersonRepository::class, $personRepoProphecy->reveal());
+
+        $request = $this->buildRequest([
+            'first_name' => $person->first_name,
+            'last_name' => $person->last_name,
+            'raw_password' => mb_substr($person->raw_password, 0, 9), // 1 below minimum 10 characters
+            'email_address' => $person->email_address,
+            'captcha_code' => 'good response',
+        ]);
+
+        $response = $app->handle($request);
+        $payloadJSON = (string) $response->getBody();
+
+        $this->assertEquals(400, $response->getStatusCode());
+        $this->assertJson($payloadJSON);
+
+        $expectedJSON = json_encode([
+            'error' => [
+                'description' => 'Validation error: Password of 10 or more characters is required to create an account',
                 'type' => 'BAD_REQUEST',
             ],
```

Comment:
@NoelLH should we also add a test to cover the edge case of white spaces in passwords? e.g. 10 spaces?

Reply:
@alihejazi In my opinion, no. Spaces are valid characters and we shouldn't build in new constraints we didn't specify.